lib/unicore/mktables

```diff
@@ -7740,8 +7709,6 @@ sub trace { return main::trace(@_); }
 
                         return if $self->carp_if_locked;
 
-                        my $addr = do { no overloading; pack 'J', $self; };
```

Comment:
This revision is different from all the other revisions in this file, as here we are removing assignment to a variable which we never subsequently use.  Perhaps, for clarity, we could remove this in a separate, previous commit?

Reply:
Ignore previous comment.  Once I looked at the p.r. *per commit* rather than as a whole, I saw that you found a few other instances of unused variables.